finder/tagged_test.go

```diff
@@ -20,6 +20,7 @@ func TestTaggedWhere(t *testing.T) {
 		{"seriesByTag('key=value')", "Tag1='key=value'", "", false},
 		{"seriesByTag('name=rps')", "Tag1='__name__=rps'", "", false},
 		{"seriesByTag('name=~cpu.usage')", "Tag1 LIKE '\\\\_\\\\_name\\\\_\\\\_=cpu%' AND match(Tag1, '^__name__=cpu.usage$')", "Tag1 LIKE '\\\\_\\\\_name\\\\_\\\\_=cpu%' AND match(Tag1, '^__name__=cpu.usage$')", false},
+		{"seriesByTag('name=~cpu|mem')", "Tag1 LIKE '\\\\_\\\\_name\\\\_\\\\_=%' AND match(Tag1, '^__name__=cpu|mem$')", "Tag1 LIKE '\\\\_\\\\_name\\\\_\\\\_=%' AND match(Tag1, '^__name__=cpu|mem$')", false},
```

Comment:
Same here. `match(Tag1, '^__name__=.*(cpu|mem).*$'` looks correct to me

Reply:
As I answer early,  this will limit the capabilities of regular expressions. We can pass `.*(cpu|mem).*` if we need manually.

Reply:
But for compability reason I can add check for ^ at the beginning/$ and the end of regex - if not found, result is a You want.